src/views/Pages/Register/Register.js

```diff
@@ -0,0 +1,241 @@
+import React, { Component } from 'react';
+import { Button, Card, CardBody, Col, Container, Form, Input, InputGroup, InputGroupAddon, InputGroupText, Row } from 'reactstrap';
+import { Nav, NavItem, NavLink, TabContent, TabPane } from 'reactstrap';
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { withRouter } from 'react-router-dom';
+import { fetchUserData } from '../../../store/actions/user'
+class Register extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      firstname: '',
+      lastname: '',
+      phonenumber: '',
+      email: '',
+      gender: 'M',
+      country: 'Nigeria',
+      dateofbirth: '',
+      formId: 'customer',
+      company: '',
+      address: '',
+      password: '',
+      loading: false,
+      activeTab: ["1"]
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  toggle(tab) {
+    this.setState({
+      activeTab: [tab]
+    });
+  }
+
+  form(formId) {
+    let optInp;
+    let u = '';
+    let f = '';
+    let { firstname, phonenumber, email, gender, country, lastname, dateofbirth, company, address } = this.state
+    // console.log(this.state)
+    let { loading, error, errorMessage, userData } = this.props.userData
+    error === 'true' && errorMessage !== '' ? (f = errorMessage) :
+      (userData.errorcount !== '0' ? (f = userData.status) : (f = ''))
+    loading === 'done' ? (u = 'Create Account') :
+      (loading === 'true' ? (u = <div id="loader"></div>) : (u = 'Create Account'))
+
+    if (formId === 'customer') {
+      optInp = <>
+              <InputGroup className="mb-3">
+          <InputGroupAddon addonType="prepend">
+            <InputGroupText><span role="img" aria-label="name">&#9892;</span></InputGroupText>
+          </InputGroupAddon>
+          <Input type="select" id="gender" value={gender} onChange={this.handleChange}>
+            <option>M</option>
+            <option>F</option>
+          </Input>
+        </InputGroup>
+        <InputGroup className="mb-4" >
+          <InputGroupAddon addonType="prepend">
+            <InputGroupText>
+              <span role="img" aria-label="name">&#128198; &nbsp; Birthday</span>
+            </InputGroupText>
+          </InputGroupAddon>
+          <Input type="date" id="dateofbirth" value={dateofbirth} onChange={this.handleChange} max="2030-12-30" min="1940-01-01" placeholder="yyyy-mm-dd" required />
+        </InputGroup>
+        <span style={{display:'block',marginBottom:'2.6em'}} ></span>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)